### DISCUSSION TITLE
How I can give the server key in desktop app?

### DISCUSSION THREAD

**Discussion by AndreeaUnity** [wazuh/wazuh#20607] [Q&A]
I can't connect my device to wazuh browser until I give my app the server key. I don't know to provide that key. I also want to make the app to take that key automatic, not with my help.

**Comment by mdiego92** [comment]
Hello @AndreeaUnity,

I suspect you're referring to the Wazuh API token, is that correct?
If it's possible, you can configure a PowerShell script using the Invoke-RestMethod. Please refer to the following documentation: https://documentation.wazuh.com/current/user-manual/api/examples.html#powershell

Hopefully, this is useful in this case.
Please let me know if you have any questions.

Regards.
